### ISSUE TITLE
Various issues from tracepot

### ISSUE THREAD

**Issue by ojacquemart** [ojacquemart/vlilleChecker#113]
Here are the most frequent issues reported by tracepot.

From high to low priority:

```
java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
	at android.os.Handler.<init>(Handler.java:200)
	at android.os.Handler.<init>(Handler.java:114)
	at android.widget.Toast$TN$2.<init>(Toast.java:336)
	at android.widget.Toast$TN.<init>(Toast.java:336)
	at android.widget.Toast.<init>(Toast.java:103)
	at android.widget.Toast.makeText(Toast.java:256)
	at com.vlille.checker.utils.ToastUtils$1.run(ToastUtils.java:34)
	at java.lang.Thread.run(Thread.java:761)
```

```
android.content.res.Resources$NotFoundException: Resource ID #0xffffffff
	at android.content.res.Resources.getValue(Resources.java:1384)
	at android.content.res.HwResources.getColor(HwResources.java:825)
	at com.vlille.checker.ui.osm.overlay.window.BubbleInfoWindow.updateTextView(BubbleInfoWindow.java:104)
	at com.vlille.checker.ui.osm.overlay.window.BubbleInfoWindow.bindStation(BubbleInfoWindow.java:98)
	at com.vlille.checker.ui.osm.overlay.window.BubbleInfoWindow.onOpen(BubbleInfoWindow.java:90)
	at com.vlille.checker.ui.osm.overlay.window.InfoWindow.open(InfoWindow.java:71)
	at com.vlille.checker.ui.osm.overlay.window.BubbleInfoWindow.open(BubbleInfoWindow.java:66)
	at com.vlille.checker.ui.osm.overlay.ItemizedOverlayWithFocus.onSingleTapUpHelper(ItemizedOverlayWithFocus.java:4141)
	at org.osmdroid.views.overlay.ItemizedIconOverlay$1.run(ItemizedIconOverlay.java:130)
	at org.osmdroid.views.overlay.ItemizedIconOverlay.activateSelectedItems(ItemizedIconOverlay.java:184)
	at org.osmdroid.views.overlay.ItemizedIconOverlay.onSingleTapConfirmed(ItemizedIconOverlay.java:123)
	at org.osmdroid.views.overlay.DefaultOverlayManager.onSingleTapConfirmed(DefaultOverlayManager.java:230)
	at org.osmdroid.views.MapView$MapViewDoubleClickListener.onSingleTapConfirmed(MapView.java:1381)
	at android.view.GestureDetector$GestureHandler.handleMessage(GestureDetector.java:303)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:150)
	at android.app.ActivityThread.main(ActivityThread.java:5621)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:794)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:684)
```

```
java.lang.NullPointerException: Attempt to invoke interface method 'java.util.Iterator java.util.List.iterator()' on a null object reference
	at com.vlille.checker.ui.osm.MapView.updateStations(MapView.java:302)
	at com.vlille.checker.ui.osm.MapView.drawLocationCircle(MapView.java:184)
	at com.vlille.checker.ui.osm.MapView.onLocationChanged(MapView.java:199)
	at android.location.LocationManager$ListenerTransport._handleMessage(LocationManager.java:300)
	at android.location.LocationManager$ListenerTransport.-wrap0(LocationManager.java)
	at android.location.LocationManager$ListenerTransport$1.handleMessage(LocationManager.java:245)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6321)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

```
java.lang.SecurityException: "passive" location provider requires ACCESS_FINE_LOCATION permission.
	at android.os.Parcel.readException(Parcel.java:1599)
	at android.os.Parcel.readException(Parcel.java:1552)
	at android.location.ILocationManager$Stub$Proxy.getLastLocation(ILocationManager.java:717)
	at android.location.LocationManager.getLastKnownLocation(LocationManager.java:1205)
	at com.vlille.checker.ui.osm.location.LocationManagerWrapper.getCurrentLocation(LocationManagerWrapper.java:1064)
	at com.vlille.checker.ui.osm.MapView.drawLocationCircle(MapView.java:3037)
	at com.vlille.checker.ui.osm.MapView.updateLocationCircle(MapView.java:144)
	at com.vlille.checker.ui.fragment.MapFragment$1.onClick(MapFragment.java:163)
	at android.view.View.performClick(View.java:5340)
	at android.view.View$PerformClick.run(View.java:21610)
	at android.os.Handler.handleCallback(Handler.java:815)
	at android.os.Handler.dispatchMessage(Handler.java:104)
	at android.os.Looper.loop(Looper.java:207)
	at android.app.ActivityThread.main(ActivityThread.java:5763)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:679)
```

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.vlille.checker/com.vlille.checker.ui.StationInfoActivity}: java.lang.NullPointerException
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2211)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2261)
	at android.app.ActivityThread.access$600(ActivityThread.java:141)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1256)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:137)
	at android.app.ActivityThread.main(ActivityThread.java:5103)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:525)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:737)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
	at com.android.internal.os.ZygoteInit.main(Native Method)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
	at com.vlille.checker.ui.StationInfoActivity.onCreate(StationInfoActivity.java:1041)
	at android.app.Activity.performCreate(Activity.java:5133)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2175)
	... 12 more
java.lang.NullPointerException
	at com.vlille.checker.ui.StationInfoActivity.onCreate(StationInfoActivity.java:1041)
	at android.app.Activity.performCreate(Activity.java:5133)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2175)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2261)
	at android.app.ActivityThread.access$600(ActivityThread.java:141)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1256)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:137)
	at android.app.ActivityThread.main(ActivityThread.java:5103)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:525)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:737)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
	at com.android.internal.os.ZygoteInit.main(Native Method)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by ojacquemart** [comment]
```
java.lang.RuntimeException: An error occured while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:300)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.IllegalStateException: java.net.UnknownHostException: Unable to resolve host "opendata.lillemetropole.fr": No address associated with hostname
	at com.vlille.checker.dataset.retrofit.VlilleClient.getStation(VlilleClient.java:48)
	at com.vlille.checker.dataset.StationRepository.getStation(StationRepository.java:70)
	at com.vlille.checker.ui.widget.WidgetAsyncTask.doInBackground(WidgetAsyncTask.java:1026)
	at android.os.AsyncTask$2.call(AsyncTask.java:288)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 4 more
Caused by: java.net.UnknownHostException: Unable to resolve host "opendata.lillemetropole.fr": No address associated with hostname
	at java.net.InetAddress.lookupHostByName(InetAddress.java:457)
	at java.net.InetAddress.getAllByNameImpl(InetAddress.java:252)
	at java.net.InetAddress.getAllByName(InetAddress.java:215)
	at okhttp3.Dns$1.lookup(Dns.java:39)
	at okhttp3.internal.connection.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:171)
	at okhttp3.internal.connection.RouteSelector.next(RouteSelector.java:2137)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:171)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:121)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:100)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:120)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.logging.HttpLoggingInterceptor.intercept(HttpLoggingInterceptor.java:212)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:185)
	at okhttp3.RealCall.execute(RealCall.java:69)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:180)
	at retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall.execute(ExecutorCallAdapterFactory.java:91)
	at com.vlille.checker.dataset.retrofit.VlilleClient.getStation(VlilleClient.java:41)
	... 8 more
Caused by: android.system.GaiException: android_getaddrinfo failed: EAI_NODATA (No address associated with hostname)
	at libcore.io.Posix.android_getaddrinfo(Native Method)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:55)
	at java.net.InetAddress.lookupHostByName(InetAddress.java:438)
	... 35 more
java.lang.IllegalStateException: java.net.UnknownHostException: Unable to resolve host "opendata.lillemetropole.fr": No address associated with hostname
	at com.vlille.checker.dataset.retrofit.VlilleClient.getStation(VlilleClient.java:48)
	at com.vlille.checker.dataset.StationRepository.getStation(StationRepository.java:70)
	at com.vlille.checker.ui.widget.WidgetAsyncTask.doInBackground(WidgetAsyncTask.java:1026)
	at android.os.AsyncTask$2.call(AsyncTask.java:288)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.net.UnknownHostException: Unable to resolve host "opendata.lillemetropole.fr": No address associated with hostname
	at java.net.InetAddress.lookupHostByName(InetAddress.java:457)
	at java.net.InetAddress.getAllByNameImpl(InetAddress.java:252)
	at java.net.InetAddress.getAllByName(InetAddress.java:215)
	at okhttp3.Dns$1.lookup(Dns.java:39)
	at okhttp3.internal.connection.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:171)
	at okhttp3.internal.connection.RouteSelector.next(RouteSelector.java:2137)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:171)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:121)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:100)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:120)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.logging.HttpLoggingInterceptor.intercept(HttpLoggingInterceptor.java:212)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:185)
	at okhttp3.RealCall.execute(RealCall.java:69)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:180)
	at retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall.execute(ExecutorCallAdapterFactory.java:91)
	at com.vlille.checker.dataset.retrofit.VlilleClient.getStation(VlilleClient.java:41)
	... 8 more
Caused by: android.system.GaiException: android_getaddrinfo failed: EAI_NODATA (No address associated with hostname)
	at libcore.io.Posix.android_getaddrinfo(Native Method)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:55)
	at java.net.InetAddress.lookupHostByName(InetAddress.java:438)
	... 35 more
java.net.UnknownHostException: Unable to resolve host "opendata.lillemetropole.fr": No address associated with hostname
	at java.net.InetAddress.lookupHostByName(InetAddress.java:457)
	at java.net.InetAddress.getAllByNameImpl(InetAddress.java:252)
	at java.net.InetAddress.getAllByName(InetAddress.java:215)
	at okhttp3.Dns$1.lookup(Dns.java:39)
	at okhttp3.internal.connection.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:171)
	at okhttp3.internal.connection.RouteSelector.next(RouteSelector.java:2137)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:171)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:121)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:100)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:120)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.logging.HttpLoggingInterceptor.intercept(HttpLoggingInterceptor.java:212)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:185)
	at okhttp3.RealCall.execute(RealCall.java:69)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:180)
	at retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall.execute(ExecutorCallAdapterFactory.java:91)
	at com.vlille.checker.dataset.retrofit.VlilleClient.getStation(VlilleClient.java:41)
	at com.vlille.checker.dataset.StationRepository.getStation(StationRepository.java:70)
	at com.vlille.checker.ui.widget.WidgetAsyncTask.doInBackground(WidgetAsyncTask.java:1026)
	at android.os.AsyncTask$2.call(AsyncTask.java:288)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: android.system.GaiException: android_getaddrinfo failed: EAI_NODATA (No address associated with hostname)
	at libcore.io.Posix.android_getaddrinfo(Native Method)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:55)
	at java.net.InetAddress.lookupHostByName(InetAddress.java:438)
	... 35 more
android.system.GaiException: android_getaddrinfo failed: EAI_NODATA (No address associated with hostname)
	at libcore.io.Posix.android_getaddrinfo(Native Method)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:55)
	at java.net.InetAddress.lookupHostByName(InetAddress.java:438)
	at java.net.InetAddress.getAllByNameImpl(InetAddress.java:252)
	at java.net.InetAddress.getAllByName(InetAddress.java:215)
	at okhttp3.Dns$1.lookup(Dns.java:39)
	at okhttp3.internal.connection.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:171)
	at okhttp3.internal.connection.RouteSelector.next(RouteSelector.java:2137)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:171)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:121)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:100)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:120)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.logging.HttpLoggingInterceptor.intercept(HttpLoggingInterceptor.java:212)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:185)
	at okhttp3.RealCall.execute(RealCall.java:69)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:180)
	at retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall.execute(ExecutorCallAdapterFactory.java:91)
	at com.vlille.checker.dataset.retrofit.VlilleClient.getStation(VlilleClient.java:41)
	at com.vlille.checker.dataset.StationRepository.getStation(StationRepository.java:70)
	at com.vlille.checker.ui.widget.WidgetAsyncTask.doInBackground(WidgetAsyncTask.java:1026)
	at android.os.AsyncTask$2.call(AsyncTask.java:288)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
```

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:330)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:255)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
	at java.lang.Thread.run(Thread.java:776)
Caused by: java.lang.IllegalStateException: java.net.SocketTimeoutException: timeout
	at com.vlille.checker.dataset.retrofit.VlilleClient.getStations(VlilleClient.java:32)
	at com.vlille.checker.dataset.StationRepository.fillStationsCache(StationRepository.java:2039)
	at com.vlille.checker.ui.async.AbstractStationsAsyncTask.doInBackground(AbstractStationsAsyncTask.java:1052)
	at android.os.AsyncTask$2.call(AsyncTask.java:316)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 4 more
Caused by: java.net.SocketTimeoutException: timeout
	at okhttp3.internal.http2.Http2Stream$StreamTimeout.newTimeoutException(Http2Stream.java:593)
	at okhttp3.internal.http2.Http2Stream$StreamTimeout.exitAndThrowIfTimedOut(Http2Stream.java:601)
	at okhttp3.internal.http2.Http2Stream.takeResponseHeaders(Http2Stream.java:146)
	at okhttp3.internal.http2.Http2Codec.readResponseHeaders(Http2Codec.java:120)
	at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:75)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.i
```

**Comment by ojacquemart** [comment]
Fixes released in the 4.2.7
